nevergrad/functions/cycling/cycling.py

```diff
@@ -0,0 +1,159 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Creator: Ryan Kroon, Email: rkroon19@gmail.com
+# Accompanying paper: Extending Nevergrad, an Optimisation Platform (in directory)
+
+# Evolving Pacing Strategies for Team Pursuit Track Cycling
+# Markus Wagner, Jareth Day, Diora Jordan, Trent Kroeger, Frank Neumann
+# Advances in Metaheuristics, Vol. 53, Springer, 2013.
+# https://link.springer.com/chapter/10.1007/978-1-4614-6322-1_4
+# Java code: https://cs.adelaide.edu.au/~markus/pub/TeamPursuit.zip
+
+from .womensteampursuit import womensteampursuit
+from .mensteampursuit import mensteampursuit
+import random
+import numpy as np
+import nevergrad as ng
+from nevergrad.parametrization import parameter as p
+from .. import base
+
+class cycling(base.ExperimentFunction):

```

Comment:
for consistency and as of pep8, please follow the [CapWords convention](https://www.python.org/dev/peps/pep-0008/#class-names) for naming classes, in this case: `Cycling`, but that's also true for all the other class names